content/blog/stratospheric-1.10-release.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: blog
+title: "Stratospheric 1.10 Release 🥳"
+description: "Release notes for the Stratospheric 1.10 ebook release"
+author: philip
+publishdate: "2023-02-21"
+featureImage: cloud-rocket-launch-three
+toc: true
+slug: stratospheric-1.10-release
+---
+
+It's time for a new Stratospheric eBook release 🥳
+
+This time it's a rather big release, as we've migrated the Stratospheric sample Todo application to Spring Boot 3.0 and Spring Cloud 3.0.
+
+The release of Spring Boot 3.0 marks a new era of developing applications with the Spring Framework. 4.5 years after the release of Spring Boot 2.0, we're now looking into a bright future and can make the most of Spring Framework 6, Java 17, and GraalVM.
+
+Even though [Spring Boot 3.0 went GA in November 2022](https://spring.io/blog/2022/11/24/spring-boot-3-0-goes-ga), it took us some time to incorporate the changes and wait for a compatible Spring Cloud AWS version. With the release of Spring Cloud AWS 3.0.0-RC1 we doubled down on the migration and wanted to provide a Spring Boot 3.0 compatible version of the ebook as soon as possible.
+
+While you may want to wait until a GA version of Spring Cloud AWS 3.0 is released, our main goals is to move fast so that you have a blueprint for the migration path. Please note that things might still change, and we're closely following the development and will update to a GA version as soon as it's available.
```

Comment:
```suggestion
While you may want to wait until a GA version of Spring Cloud AWS 3.0 is released, our main goals is to move fast so that you have a blueprint for the migration path. Please note that things might still change, and we're closely following the development and will update to a GA version as soon as it is available.
```